tools/generator/test/GeneratorTests.swift

```diff
@@ -340,6 +340,7 @@ final class GeneratorTests: XCTestCase {
         func createFilesAndGroups(
             in pbxProj: PBXProj,
             buildMode: BuildMode,
+            _forFixtures: Bool,
```

Comment:
Where is this used? I'd expect a change at the call site in this PR, but don't see any. Maybe there's some magic happening that's eliding the parameter label at the call site for some reason?

Reply:
This `createFilesAndGroups()` is called inside of `generate()`, where it is passed the constant `false` down below. I didn't feel it was worth tracking, so here and the previous spot I just add the parameter as needed, but I don't track it in the `XYZCalled` structs.